app/helpers/common_parameters_helper.rb

```diff
@@ -16,17 +16,36 @@ def parameter_value_field(value)
   end
 
   def parameter_value_content(id, value, options)
-    content_tag(:span, options[:popover], :class => "input-group-addon") +
-      text_area_tag(id, value, { :rows => 1,
-                                 :class => 'form-control no-stretch',
-                                 :'data-property' => 'value',
-                                 :'data-hidden-value' => Parameter.hidden_value,
-                                 :'data-inherited-value' => options[:inherited_value],
-                                 :name => options[:name].to_s,
-                                 :disabled => options[:disabled] })
+    lookup_key = options[:lookup_key]
+    option_hash = { :rows => 1,
+                    :class => 'form-control no-stretch',
+                    :'data-property' => 'value',
+                    :'data-hidden-value' => LookupKey.hidden_value,
+                    :'data-inherited-value' => options[:inherited_value],
+                    :name => options[:name].to_s,
+                    :disabled => options[:disabled] }
+    if lookup_key.present? && lookup_key.hidden_value?
+      text_area = password_field_tag(id, value, option_hash)
```

Comment:
I'd name it field or something, as it can be a password field, in which case, the name "text_area" is, well, odd.